demo/tests/test_perfs_rec.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+import sys
+import os
+import json
+
+from django.utils.version import get_complete_version
+from django.test import TestCase as OrigTestCase, Client
+
+try:
+    from django.urls import reverse
+except ImportError:
+    from django.core.urlresolvers import reverse
+
+from django_perf_rec import TestCaseMixin
+
+
+class TestCase(TestCaseMixin, OrigTestCase):
+    """
+    Add use perf rec mixin
+    """
+
+    c = Client()
+    dir_path = os.path.dirname(os.path.realpath(__file__))
+    yml_dir = 'perfs'
+    file_name_pattern = 'django%s-py%s.yml'
+
+    def get_yml_dir_path(self):
+        """
+        Function to generate a path for all the Python and Django version
+        """
+        python_version = '%s%s' % (
+            sys.version_info.major, sys.version_info.minor
+        )
+        django_version = '%s%s' % get_complete_version()[:2]
+        file_name = self.file_name_pattern % (django_version, python_version)
+
+        return os.path.join(self.dir_path, self.yml_dir, file_name)
+
+    def record_performance(self, record_name=None, path=None):
+        if path is None:
+            path = self.get_yml_dir_path()
+
+        return super(TestCase, self).record_performance(
+            path=path,
+            record_name=record_name
+        )
+
+
+class TestPerfRec(TestCase):
+    """
+    """
+    fixtures_folder = 'fixtures'
+
+    def setUp(self):
```

Comment:
You can use `setUpClass()` classmethod, to load files only once. See warning section on https://docs.djangoproject.com/en/1.8/topics/testing/tools/#simpletestcase